### ISSUE TITLE
How to enable indexing for entire project ?

### ISSUE THREAD

**Issue by tbm98** [Dart-Code/Dart-Code#3363]
My case: 
when I open a project, it only indexing files that are opening, so when I click to open other file, it start index very slow.

**Comment by tbm98** [comment]
please see this video.

https://user-images.githubusercontent.com/52562340/119650063-95ce0c00-be4d-11eb-8202-8f77bb630695.mov

**Comment by DanTup** [comment]
Can you confirm what version of Flutter you're using?

**Comment by DanTup** [comment]
@tbm98 sorry, I meant the Flutter SDK not the Flutter extension. This seems like a bug I would have expected to be fixed in Flutter 2.2.

**Comment by tbm98** [comment]
Yes. flutter sdk 1.22.2
I'll try again with 2.2

**Comment by DanTup** [comment]
Oh yes, I mixed up the versions myself 😄

But yes, I think you're likely seeing an issue that was fixed by https://github.com/dart-lang/sdk/commit/ddd70f5cd0275235f27c00582da4f7fdadc9da8d. I'll close this because I'm fairly certain it's fixed, but let me know if you still see it on Flutter 2.2 (be sure to restart VS Code after upgrading to make sure the analysis server is restarted) and we can re-open and capture some logs. Thanks!
